From 310e4b64d3116600c1bc085ac750098ec111141f Mon Sep 17 00:00:00 2001
From: Jakob Haufe <sur5r@sur5r.net>
Date: Thu, 10 May 2012 20:26:48 +0200
Subject: [PATCH 09/12] libv4lcontrol: Add Lenovo Thinkpad X220 Tablet to
 upside down table

Signed-off-by: Gregor Jasny <gjasny@googlemail.com>
(cherry picked from commit c8dc32601812326646e4e8fd1388a0cc9d2a512c)
---
 lib/libv4lconvert/control/libv4lcontrol.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/lib/libv4lconvert/control/libv4lcontrol.c b/lib/libv4lconvert/control/libv4lcontrol.c
index 3648d91..8795e2f 100644
--- a/lib/libv4lconvert/control/libv4lcontrol.c
+++ b/lib/libv4lconvert/control/libv4lcontrol.c
@@ -132,6 +132,8 @@ static const struct v4lcontrol_flags_info v4lcontrol_flags[] = {
 	{ 0x04f2, 0xb213, 0, "FUJITSU", "FJNBB19",
 		V4LCONTROL_HFLIPPED | V4LCONTROL_VFLIPPED, 0,
 		"FUJITSU", "LIFEBOOK NH751" },
+	{ 0x04f2, 0xb217, 0, "LENOVO", "42982YG",
+		V4LCONTROL_HFLIPPED | V4LCONTROL_VFLIPPED },
 	{ 0x04f2, 0xb217, 0, "LENOVO", "42992QG",
 		V4LCONTROL_HFLIPPED | V4LCONTROL_VFLIPPED },
 	{ 0x04f2, 0xb27c, 0, "LENOVO", "12973MG",
-- 
1.7.10

